templates/nutrition_facts_table.tt.html

```diff
@@ -47,13 +47,13 @@
 [% FOREACH table IN tables %]
 
 	<table id="[% table.id %]_data_table" class="data_table">
-
+    <caption>Nutrition Facta Table</caption>
```

Comment:
small typo: Facta instead of Facts. But this should be internationalized, using lang("nutrition_data")